index.js

```diff
@@ -30,6 +30,7 @@ program
   .argument('[files...]', 'a glob of files to to test the EcmaScript version against')
   .option('--module', 'use ES modules')
   .option('--allow-hash-bang', 'if the code starts with #! treat it as a comment')
+  .option('--files <files>', 'a glob of files to to test the EcmaScript version against (alias for [files...])')
```

Comment:
Didn't include the `-f` short flag from the original proposal since it didn't seem to fit with the style of the other flags.